svc-api/handle/handlers.go

```diff
@@ -535,6 +535,12 @@ func GetMetadata(ctx iris.Context) {
 					models.Include{Namespace: "FabricCollection"},
 				},
 			},
+			models.Reference{URI: "http://redfish.dmtf.org/schemas/v1/NetworkAdapter_v1.xml",
+				TopInclude: []models.Include{
+					models.Include{Namespace: "NetworkAdapter"},
+					models.Include{Namespace: "NetworkAdapter.v1_5_0"},
+				},
+			},
 		},
```

Comment:
include Namespace="NetworkAdapterCollection" as well

Reply:
'NetworkAdapterCollection' is already present (line 443)

Reply:
OK add Namespace="NetworkAdapter.v1_2_0" Namespace="NetworkAdapter.v1_3_0" Namespace="NetworkAdapter.v1_4_0" to NetworkAdapter

Reply:
done